projects/plugins/jetpack/to-test.md

```diff
@@ -1,48 +1,54 @@
+### Payments Buttons
 
+A new Payments Buttons block was added that acts as a container for several Payment Button blocks [[#25297](https://github.com/Automattic/jetpack/pull/25297)]. Note that testing this requires a pid plan and Stripe connection.
 
-### Jetpack Connection
+- On a Jetpack connected site, create a new post and add a Payments Buttons block.
+- If you don't have a paid plan, you will be asked to upgrade.
+- You will be asked to connect to Stripe.
+- Select the Payment Buttons block and try adding several Payment Button block inside it.
+- Make sure you can select different plans for each button.
+- Make sure you can change the width of each button.
+- View the post and confirm Payment Blocks look as intended.
 
-Over the past month, various issues related to the Jetpack connection flow have had fixes. Below are several different connection scenarios. If you can test some, all, or any one of these (or others not mentioned here) that would be really helpful.
 
-- With Jetpack installed and active on your test site (but not connected), try different connection flows. If you have access to the Beta plugin settings on your site, double check that the Release Candidate is active.
-	- Some connection flows include via the full-page connection banner that you see immediately after activating the plugin, the connection banner on the plugins page or wp-admin dashboard, and the connection banner on the Jetpack dashboard (`wp-admin/admin.php?page=jetpack#/`)
-- [This flow currently doesn't seem to work when testing with the Jetpack Beta Tester plugin, so unless you have access to the site's files via FTP/SSH to copy over the changes in `class.jetpack.php` [here](https://github.com/Automattic/jetpack/pull/25135) it will be difficult to test]. With Jetpack installed and active on your test site (but not connected), install and activate WooCommerce and begin to go through the onboarding flow (no need to add payment details - you can select the 'cash on delivery' option to keep going through the setup flow). You should get as far as activating a theme, and then should be redirected to a Jetpack authorization prompt.
-- Using the WordPress app, under 'My Sites' you can click the plus icon to add a new, self-hosted site. Continue to add a site, and once added go to Plugins > Jetpack, then scroll down and select 'Set Up Jetpack'. While the mobile connection flows may differ slightly and some issues may not be on the Jetpack side - there is currently an issue connecting Jetpack via the Stats menu item for example.
-- Any other connection or onboarding flows that connect to Jetpack, that you are aware of.
+### Google documents, spreadsheets and slideshows
+
+We are adding support for Google documents, spreadsheets and slideshows. [[#24628](https://github.com/Automattic/jetpack/pull/24628)]. Please note these blocks are still in beta so testing them requires enabling JETPACK_BETA_BLOCKS constant in the site Settings. To test, try the following:
```

Comment:
Enabling that constant via site settings is only possible on JN sites, but would have to be added in wp-config.php otherwise with `define( 'JETPACK_BETA_BLOCKS', true );` - we could add mention of that?